src/App.js

```diff
@@ -1,11 +1,57 @@
 import "./App.css";
+import Header from "./components/Header.js";
+import ToDoList from "./components/ToDoList.js";
+import { useState } from "react";
+
+// const toDoArray = [
+//   { name: "shoppingItem", status: "Pending" },
+//   { name: "running", status: "Done" },
+// ];
+
+export default function App() {
+  const [toDo, setToDo] = useState([]);
+
+  function handleAddToDo(toDoItem) {
+    const newToDo = [
+      ...toDo,
+      {
+        name: toDoItem,
+        status: false,
+      },
+    ];
+    setToDo(newToDo);
+  }
+
+  function handleDeleteToDo(name) {
+    const removedToDo = toDo.filter((toDo) => toDo.name !== name);
+    setToDo(removedToDo);
+  }
+
+  function togglePendingButton(buttonContent) {
```

Comment:
I'd rename the parameter `buttonContent` to something more meaningful, like `todoName` or `name`